README.md

```diff
@@ -166,6 +166,21 @@ Time Zone can be set by using `-Duser.timezone` system property, e.g. `-Duser.ti
 ## Statistics information
 If you want to know how TiSpark could benefit from TiDB's statistic information, read more [here](./docs/userguide.md).
 
+## Compatibility with tidb-3.0
+### View
+TiDB start to support `view` from `tidb-3.0`. 
+
+TiSpark currently does not support `view` and will filter all views from catalog. Users cannot select data from views through TiSpark.
+
+### Table Partition
+`tidb-3.0` supports both `Range Partition` and `Hash Partition`. 
+
+TiSpark currently does support `Range Partition` and `Hash Partition`. Users can select data from `Range Partition` table and `Hash Partition` table through TiSpark. 
+
+In most cases TiSpark will use full table scan. Only in such cases TiSpark will apply partition pruning:
```

Comment:
s/Only in such cases TiSpark will apply partition pruning/Only in the following cases TiSpark will apply partition pruning to build a better plan